Automattic/WP-Job-Manager
# Localize jQuery datepicker on WP 4.6+

Fixes #767

#### Changes proposed in this Pull Request:

* Since the radical `wp_localize_jquery_ui_datepicker()` function, which localizes jQuery UI's datepicker, was put into core in WP 4.6, jQuery's `datePicker` was already being localized using WP's date settings.
* Update all other translated date formats to use the WordPress date format option.

#### Testing instructions:

* Using various WP date formats settings, verify saving the job expiration date in WP admin still works.
* Clear the expiration date and verify it saves. If the setting to have a default expiration date is set, the date will be automatically set (default +30 days). If that setting is cleared, the expiration date should remain empty.
* Verify dates in past also change the post status to expired. 


#### Proposed changelog entry for your changes:
* Change:  Use WordPress' date format settings on job expiration field in WP admin.
